src/adapter.js

```diff
@@ -144,10 +144,8 @@ const adapter = {
                 let callbacks = map.callbacks
 
                 if (callbacks[keypath]) {
-                  callbacks[keypath].slice().forEach(callback => {
-                    if (callbacks[keypath].indexOf(callback) > -1) {
-                      callback()
-                    }
+                  callbacks[keypath].forEach(cb => {
+                      cb()
```

Comment:
The version with {} generates smaller code:

http://babeljs.io/repl/#?experimental=false&evaluate=true&loose=false&spec=false&code=array.forEach%28%28cb%29%3D%3Ecb%28%29%29%0Aarray.forEach%28%28cb%29%3D%3E{%0A%20%20cb%28%29%0A}%29